src/app/containers/Image/index.jsx

```diff
@@ -4,6 +4,7 @@ import { imageModelPropTypes } from '../../models/propTypes/image';
 import Figure from '../Figure';
 
 const DEFAULT_IMAGE_RES = 640;
+const srcSetValues = [360, 480, 600, 720, 840, 1024, 2048];
```

Comment:
Values pending discussion with designer on breakpoints, right? (Also, one of these ought to match the `DEFAULT_IMAGE_RES`, I would think 🤔 )

Reply:
Agreed, these are for investigation only